cmd/robot/subject.go

```diff
@@ -48,6 +48,7 @@ func init() {
 
 type subjectUploadVerb struct {
 	RobotOptions
+	API       string        `help:"the api to capture, can be either gles or vulkan (default:gles)"`
```

Comment:
The flag system [does support enums](https://github.com/google/gapid/blob/e873228976e27d96964158506308962bed7e7db8/core/app/flags/choices.go#L34-L41).

Reply:
I was following the pattern that gapit uses for accepting the API seen in cmd/gapit/flags.go. Should I change that instance as well to be an enum?

Reply:
You can do, but you don't have to. That flag might become a list, in which case keeping-as-is is fine.

Reply:
ok fixed.